test/stripe/account_test.rb

```diff
@@ -78,5 +78,41 @@ class AccountTest < Test::Unit::TestCase
         Stripe::Account.retrieve(:api_key => nil)
       end
     end
+
+    should "be able to create a bank account" do
+      resp = {
+        :id => 'acct_1234',
+        :external_accounts => {
+          :object => "list",
+          :url => "/v1/accounts/acct_1234/external_accounts",
+          :data => [],
+        }
+      }
+      @mock.expects(:get).once.returns(test_response(resp))
+      a = Stripe::Account.retrieve
+
+      @mock.expects(:post).
+        once.
+        with('https://api.stripe.com/v1/accounts/acct_1234/external_accounts', nil, 'external_account=batok_1234').
+        returns(test_response(resp))
+      a.external_accounts.create({:external_account => 'batok_1234'})
```

Comment:
nit: I know it's just fake data, but the prefix here should be `btok` not `batok`